Source/Details/_ASDisplayViewAccessiblity.mm

```diff
@@ -169,8 +174,20 @@ static void CollectAccessibilityElementsForContainer(ASDisplayNode *container, _
   }
 
   SortAccessibilityElements(labeledNodes);
-  NSArray *labels = [labeledNodes valueForKey:@"accessibilityLabel"];
-  accessiblityElement.accessibilityLabel = [labels componentsJoinedByString:@", "];
+
+  if (@available(iOS 11.0, *)) {
+    NSArray *attributedLabels = [labeledNodes valueForKey:@"accessibilityAttributedLabel"];
+    NSMutableAttributedString *attributedLabel = [NSMutableAttributedString new];
+    [attributedLabels enumerateObjectsUsingBlock:^(id  _Nonnull obj, NSUInteger idx, BOOL * _Nonnull stop) {
+      if (idx != 0)
```

Comment:
The convention is to use curly brackets regardless of the number of statements followed.

Reply:
Sorry I thought I followed convention because I saw this in _PendingState.mm a lot for single statements (L375, L377), but I'll change my code to use curly braces.

Reply:
Yeah, we need to have it written down in `CONTRIBUTING.md`. Others may disagree, but I personally follow this convention (as much as possible).

Reply:
I wouldn't change it for `applyToView:withSpecialPropertiesHandling:` though since every flag check seems to skip the curly braces